com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -130,11 +130,11 @@ public static void RegisterNotificationChannel(AndroidNotificationChannel channe
             }
             else if (string.IsNullOrEmpty(channel.id))
```

Comment:
Here should check `channel.Name`... 
Would you like to fix it at your side or let me fix from the main repo?

Reply:
I'll merge anyway. Thanks.